### PR TITLE
Fab 1564

### PR DESCRIPTION
**Pull request by ljha-CS**

### CODE REVIEW

File: src/commands/deploy.js [hunk 1]
```diff
@@ -160,7 +158,7 @@ module.exports.DeployCampaignCommand = class {
         const filepaths = {};
         const promises = [];
 
-        yauzl.open('x.zip', { lazyEntries: true }, (err, zipfile) => {
+        yauzl.open(`${campaignName}.zip`, { lazyEntries: true }, (err, zipfile) => {
```
Comment (yreddy-CS) [inline src/commands/deploy.js:161]: Wondering why this change is there... It was fixed as part of https://github.com/CognitiveScale/cortex-cli/pull/377/files

**Review by yreddy-CS — APPROVED** [review]

**Review by apathuri-CS — APPROVED** [review]

File: src/commands/deploy.js (hunk 1 above)
Comment (ljha-CS) [reply src/commands/deploy.js:161]: local testing

**Review by akprasad-cs — APPROVED** [review]
